src/backend/TrafficCourts/Staff.Service/Services/DisputeService.cs

```diff
@@ -51,20 +56,36 @@ public async Task<Dispute> UpdateDisputeAsync(Guid disputeId, Dispute dispute, S
 
     public async Task CancelDisputeAsync(Guid disputeId, CancellationToken cancellationToken)
     {
+        _logger.LogDebug("Dispute cancelled.");

```

Comment:
log messages dont need periods. Consider them a phrase, not a complete sentence.

Reply:
Done